Documentation/ceph-cluster-crd.md

```diff
@@ -1319,128 +1319,63 @@ The features available from the external cluster will vary depending on the vers
 
-**Example**:
-
-```console
-ceph osd pool ls
-```
->```
->my-store.rgw.control
->my-store.rgw.meta
->my-store.rgw.log
->my-store.rgw.buckets.index
->my-store.rgw.buckets.non-ec
->my-store.rgw.buckets.data
->```
+In order to configure an external Ceph cluster with Rook, we need to inject some information in order to connect to that cluster.
 
-In this example, you can simply export RGW_POOL_PREFIX before executing the script like this:
+1. Run the python script [create-external-cluster-resources.py](/deploy/examples/create-external-cluster-resources.py) for creating all users and keys.
 
 ```console
-export RGW_POOL_PREFIX=my-store
+python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --namespace <namespace> --format bash
 ```
 
-The script will automatically create users and keys with the lowest possible privileges and populate the necessary environment variables for `deploy/examples/import-external-cluster.sh` to work correctly.
-
-Finally, you can simply execute the script like this from a machine that has access to your Kubernetes cluster:
-
+rbd-data-pool-name should be created, and if you need to have CephFS secret details you have to pass filesystem name, and if you need to create rgw_admin_ops_user you need to pass rgw_endpoint (Rados GateWay endpoint (in <IP>:<PORT> format)) , and if you need to export rgw-pool-prefix prepare rgw pools
+example: 
 ```console
-bash deploy/examples/import-external-cluster.sh
+python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --cephfs-filesystem-name <filesystem-name> --rgw-endpoint  <rgw-endpoint> --namespace <namespace> --rgw-pool-prefix <rgw-pool-prefix> --format bash
 ```
 
-#### CephCluster example (consumer)
+>Note: Use format bash, as to keep up with the further steps, and rbd-data-pool and cephfs-filesystem, should be created in advance.
 
-Assuming the above section has successfully completed, here is a CR example:
+2. After that you need to copy the bash output.
 
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephCluster
-metadata:
-  name: rook-ceph-external
-  namespace: rook-ceph-external
-spec:
-  external:
-    enable: true
-  crashCollector:
-    disable: true
-  # optionally, the ceph-mgr IP address can be pass to gather metric from the prometheus exporter
-  #monitoring:
-    #enabled: true
-    #rulesNamespace: rook-ceph
-    #externalMgrEndpoints:
-      #- ip: 192.168.39.182
-    #externalMgrPrometheusPort: 9283
-```
+#### Run commands on machine that has access to your Kubernetes cluster:
 
-Choose the namespace carefully, if you have an existing cluster managed by Rook, you have likely already injected `common.yaml`.
-Additionally, you now need to inject `common-external.yaml` too.
+1. Deploy Rook-Ceph, create [common.yaml](/deploy/examples/common.yaml), [crds.yaml](/deploy/examples/crds.yaml) and [operator.yaml](/deploy/examples/operator.yaml) manifests.
```

Comment:
We just need common-external.yaml, right? Not common.yaml?
```suggestion
1. Deploy Rook-Ceph, create [common-external.yaml](/deploy/examples/common-external.yaml), [crds.yaml](/deploy/examples/crds.yaml) and [operator.yaml](/deploy/examples/operator.yaml) manifests.
```

Reply:
Can we deploy without common.yaml? Or is that a follow-up PR?

Reply:
Need to do some check-in live cluster, yaa can re-factor in the follow-up PR.